lib/minify.js

```diff
@@ -36,6 +36,7 @@ function minify(files, options) {
         options = defaults(options, {
             compress: {},
             ie8: false,
+            safari10: true,
```

Comment:
I just thought that safari10 option should be enabled default because it is latest version. so I made changes -you commented previous- in same reason.
if default should be `false`, other changes are also not needed.
I'll change it to `false`